### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Dhihram** [epiforecasts/EpiNow2#342] [Installation]
I found this when I run the library of EpiNow2:
Error: package or namespace load failed for ‘EpiNow2’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 namespace ‘cli’ 3.3.0 is already loaded, but >= 3.4.0 is required
is there any solution related to this?

**Comment by sbfnk** [comment]
Hi, it should be possible to resolve this by re-starting R, possibly after running `install.packages("cli")`.
